Marlin/Configuration.h

```diff
@@ -1498,6 +1498,37 @@
 //
 //#define OLED_PANEL_TINYBOY2
 
+//
+// Makeboard 3D Printer Parts 3D Printer Mini Display 1602 Mini Controller
+// https://www.aliexpress.com/item/Micromake-Makeboard-3D-Printer-Parts-3D-Printer-Mini-Display-1602-Mini-Controller-Compatible-with-Ramps-1/32765887917.html
+//
+// 2 line display.  
```

Comment:
if you must add this info, add it to the description line:

`// Makeboard 3D Printer Parts 3D Printer Mini Display 1602 Mini Controller 2 line display`

and delete extra blank line